tests/prague/eip7685_general_purpose_el_requests/test_multi_type_requests.py

```diff
@@ -116,12 +125,12 @@ def get_permutations(n: int = 3) -> Generator[Any, None, None]:
         ),
     ]
     for perm in permutations(requests, n):
-        yield pytest.param([p[1] for p in perm], id="+".join([p[0] for p in perm]))
+        yield pytest.param([p[1] for p in perm], id="+".join([p[0] for p in perm]))  # type: ignore
```

Comment:
```suggestion
        yield pytest.param([p[1] for p in perm], id="+".join([p[0] for p in perm]))
```
Only if the previous comment is applied.